Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -1210,10 +1228,18 @@ static void webkit_web_view_class_init(WebKitWebViewClass* webViewClass)
      * See webkit_web_view_get_favicon() for more details.
      */
     sObjProperties[PROP_FAVICON] =
+#if USE(GTK4)
+        g_param_spec_object(
+            "favicon",
+            nullptr, nullptr,
+            GDK_TYPE_TEXTURE,
+            WEBKIT_PARAM_READABLE);
+#else
         g_param_spec_pointer(
             "favicon",
             nullptr, nullptr,
             WEBKIT_PARAM_READABLE);
+#endif
```

Comment:
I wonder if gir-scanner will be able to handle this. We know it cannot handle conditional compilation for signals, so I suspect it will trip over properties as well. Probably should split out into the WebKitWebViewGtk3.cpp and WebKitWebViewGtk4.cpp?

Reply:
I think the problem is conditional compilation of the documentation comment, not the definition itself. In this case the doc is the same.